crates/wash-cli/tests/wash_build.rs

```diff
@@ -540,6 +540,79 @@ async fn integration_build_handles_dashed_names() -> Result<()> {
     Ok(())
 }
 
+/// Ensure that wash build can handle absolute and relative paths changing for the
+/// project directory, build directory, WIT directory, and wasmcloud.toml file.
+#[tokio::test]
+#[cfg_attr(not(can_reach_ghcr_io), ignore = "ghcr.io is not reachable")]
```

Comment:
For the template, this will use github.com right?

Reply:
Oh yes that's right. How does this even work? Is this like an external attribute we pass in during test time?